src/preload.sh

```diff
@@ -313,7 +313,7 @@ function start_docker_daemon() {
 
     # If this preload script was ran before implementing the rce/docker fix,
     # make sure you cleanup
-    if [[ -d "${APPFS_MNT}/docker" ]]; then
+    if [[ -d "${APPFS_MNT}/rce" ]]; then
```

Comment:
I think the previous version is correct, what this does is check if it's an image using `docker` and then removes any stray `rce` stuff that may have been added if running with an old version of the preloader that assumed `rce` only, and then sets the docker dir to `"${APPFS_MNT}/docker"`